src/components/SelectionList/BaseListItem.tsx

```diff
@@ -49,8 +48,8 @@ function BaseListItem({
     return (
         <OfflineWithFeedback
             onClose={() => onDismissError(item)}
-            pendingAction={item.pendingAction}
-            errors={item.errors}
+            pendingAction={isUserItem ? item.pendingAction : undefined}
+            errors={isUserItem ? item.errors : undefined}
```

Comment:
Why are we changing this?

Reply:
Because this props do not exist on type RadioItem - I think it is better to not set them in types as undefined always